components/theme/colors.tsx

```diff
@@ -1,8 +1,18 @@
-export type ColorTypes = 'white' | 'black' | 'darkgrey' | 'lightgrey'
+import styles from '../../scss/__variables.module.scss'
+
+export type ColorTypes =
+  | 'white'
+  | 'black'
+  | 'darkgrey'
+  | 'lightgrey'
+  | 'danger'
+  | 'primary'
 
 export const colors = {
   white: '#fff',
   black: '#292929',
   darkgrey: '#5d656e',
-  lightgrey: '#8e8e8e'
+  lightgrey: '#8e8e8e',
```

Comment:
These `color` constants should also be enums since we are using typescript

Reply:
Oh wait hmm this might require more refactoring, since other files may already be using this colors constant object. Might wanna leave it for another PR.